src/sql/workbench/services/notebook/common/notebookServiceImpl.ts

```diff
@@ -158,6 +170,21 @@ export class NotebookService extends Disposable implements INotebookService {
 		}
 	}
 
+	private updateSQLRegistrationWithConnectionProviders() {
+		// Update the SQL extension
+		let sqlNotebookProvider = this._providerToStandardKernels.get(notebookConstants.SQL);
+		if (sqlNotebookProvider) {
+			let sqlConnectionTypes = this._queryManagementService.getRegisteredProviders();
+			let provider = sqlNotebookProvider.find(p => p.name.toLowerCase() === notebookConstants.SQL.toLowerCase());
```

Comment:
Nit: ideally just toLowercase the `notebookConstants.SQL` value once? Or even just compare as uppercase, frankly since we own both sides we know they're the same...